OpenRA.Platforms.Default/Sdl2PlatformWindow.cs

```diff
@@ -130,7 +133,7 @@ public Sdl2PlatformWindow(Size requestEffectiveWindowSize, WindowMode windowMode
 				Console.WriteLine("Using SDL 2 with OpenGL{0} renderer", useGLES ? " ES" : "");
 
 				SDL.SDL_DisplayMode display;
-				SDL.SDL_GetCurrentDisplayMode(0, out display);
+				SDL.SDL_GetCurrentDisplayMode(videoDisplay, out display);
```

Comment:
The `SDL_GetDisplayDPI` call just below will also need to use `videoDisplay`, and the workaround for macOS on lines 243/244 will need to be removed (the problem it was working around doesn't seem to occur anymore, and it otherwise forces the game back to the left-most monitor).

Reply:
I removed the code that moves the window back to the origin on MACs.